### ISSUE TITLE
EPUB 3.3 2021-08-18

### ISSUE THREAD

**Issue by dauwhe** [w3cping/privacy-request#52]
We have conducted a self-review of our specs [EPUB 3.3](https://w3c.github.io/epub-specs/epub33/core/) and [EPUB 3.3 Reading Systems](https://w3c.github.io/epub-specs/epub33/rs/) and the results can be found at https://github.com/w3c/epub-specs/blob/main/epub33/explainers/EPUB-33-security-privacy.md.

Please check our findings.

We do not have a particular deadline for this review, although we hope to go to CR late this year. 

Note that our working group does not have many privacy experts, and EPUB is largely presented to the users outside of what we think of as the web. 

Please let us know if you have any questions!

—Dave Cramer, co-chair of EPUB 3 Working Group.

**Comment by sandandsnow** [comment]
PING is planning to discuss this review in its meeting on 16 September 2021

**Comment by dauwhe** [comment]
> PING is planning to discuss this review in its meeting on 16 September 2021

I will be happy to join the meeting if that will help!

**Comment by sandandsnow** [comment]
@dauwhe you would be most welcome - I will send you the call details.

**Comment by sandandsnow** [comment]
Review discussed on PING call on [16 September 2021](https://www.w3.org/Privacy/IG/summaries/PING-minutes-20210916)

Go to https://github.com/w3cping/tracking-issues/issues to track the progress of any issues raised as a result of the privacy review.

Issue closed as the review has been undertaken.

(Note: there may still be outstanding privacy considerations identified in the review that have not yet been resolved.)

**Comment by samuelweiler** [comment]
Forwarded from Wendy Reid.  I've pointed out that they did not request an initial review of the a11y doc, so we may get a request to review that.

> EPUB 3.3 is in the process of moving to CR, and there were some questions about
> horizontal review for i18n and Privacy and Security we were hoping you could
> confirm for us. We would like it if you could both review the changelogs for the
> three documents and confirm that everything looks ok to you, if anything needs to
> be addressed before CR, or if there are things we can address during CR. For i18n
> it is just that we had review done a year ago, and the document has changed since,
> though with no major i18n-specific changes. For Privacy and Security, we
> particularly want to make sure the Accessibility 1.1 doc is also approved. The
> changelogs for all three contain all of our substantive changes in the documents.
> 
> The changelogs for the three documents can be found here:
> 
> https://w3c.github.io/epub-specs/epub33/a11y/#change-log
> 
> https://w3c.github.io/epub-specs/epub33/core/#change-log
> 
> https://w3c.github.io/epub-specs/epub33/rs/#change-log 
> 
>  
> We ask that you give us a reply by April 28, as we are hoping to go to CR by May
> 5.

**Comment by npdoty** [comment]
Apologies, I got behind on this topic. (For a while I wasn't replying because I didn't know the group was waiting on me, but then once I heard that it was waiting on me, I was busy and didn't get to it quickly.)

I have started re-reviewing, including the changes that were made in response to previous comments, and the a11y doc. I really wanted to finish that today as requested, but didn't quite. I'll have feedback tomorrow, Friday, April 29th.
